eth/stagedsync/stage_txlookup.go

```diff
@@ -99,6 +101,11 @@ func TxLookupTransform(logPrefix string, tx kv.RwTx, startKey, endKey []byte, qu
 				return err
 			}
 		}
+
+		if err := next(k, crypto.Keccak256(append(borPrefix, blockHash[:]...)), bigNum.SetUint64(blocknum).Bytes()); err != nil {
```

Comment:
blockNumber encoding is missing there (you can use `k`), it needs to be `borPrefix | k | blockHash`